### ISSUE TITLE
[Quest][Outlands][Torgos!]

### ISSUE THREAD

**Issue by Seperest** [dalaranwow/dalaran-wow#5601]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Trachela does not drop the carcass required to summon Torgos.

**Expected behaviour**: Trachela should always drop the carcass if the player killing her has the [Torgos!] quest

**Steps to reproduce the problem**: 

1. Use your GM powers to add the quest [Torgos!] to your questlog.
2. Kill Trachela.
3. Gasp at the fact that no carcass dropped.

**Include proofs for this behaviour**
I have no idea how to prove this... Just try to kill her with quest... 

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Item ID: 30618

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Drop chance is 100%, make sure nothing was messing with your drop rate.

![image](https://user-images.githubusercontent.com/5057942/45558402-1e442400-b840-11e8-9b7e-897980d3b157.png)

Closing.
